src/users/UsersProfileScreen.js

```diff
@@ -0,0 +1,19 @@
+/* @flow strict-local */
+import React from 'react';
+import type { Node } from 'react';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+import type { AppNavigationProp } from '../nav/AppNavigator';
+import UsersProfileCard from './UsersProfileCard';
+
+type Props = $ReadOnly<{|
+  navigation: AppNavigationProp<'users'>,
+|}>;
+
+export default function UsersInfoScreen(props: Props): Node {
+  return (
+    <SafeAreaView mode="padding" edges={['top']} style={{ flex: 1 }} scrollEnabled={false}>
+      <UsersProfileCard filter="" />
+    </SafeAreaView>
+  );
```

Comment:
Instead of having a trivial wrapper component like this, let's have just one component that does the job of both this and the inner component.

(I see the existing `UsersScreen` looks this way too. Maybe I'll go and fix that now.)

Reply:
> (I see the existing `UsersScreen` looks this way too. Maybe I'll go and fix that now.)

#5540